packages/components/menu/__tests__/menu.spec.ts

```diff
@@ -3,7 +3,7 @@ import { nextTick } from 'vue'
 import { sleep } from '@element-plus/test-utils'
 import { rAF } from '@element-plus/test-utils/tick'
 
-import Menu from '../src/menu.vue'
+import Menu from '../src/menu'
```

Comment:
Why remove `.vue` extension?

Reply:
`menu.vue` has been removed to `menu.ts`

Reply:
> `menu.vue` has been removed to `menu.ts`

OK~